### PR TITLE
ifdef the exporters as specifying stricter linker flags than what's in default CMake causes linking issues

### PR DESCRIPTION
**Pull request by Nimer-88**

### CODE REVIEW

**Comment by iamAdrianIusca** [comment]
good catch!

File: code/Common/Exporter.cpp [hunk 1]
```diff
@@ -80,32 +80,61 @@ namespace Assimp {
...
 void ExportSceneXFile(const char*,IOSystem*, const aiScene*, const ExportProperties*);
+#endif
+#ifndef ASSIMP_BUILD_NO_STEP_EXPORTER
 void ExportSceneStep(const char*,IOSystem*, const aiScene*, const ExportProperties*);
+#endif
+#ifndef ASSIMP_BUILD_NO_OBJ_EXPORTER
 void ExportSceneObj(const char*,IOSystem*, const aiScene*, const ExportProperties*);
 void ExportSceneObjNoMtl(const char*,IOSystem*, const aiScene*, const ExportProperties*);
+#endif
+#ifdef ASSIMP_BUILD_NO_STL_EXPORTER
```
Comment (turol) [inline code/Common/Exporter.cpp]: This one is wrong, you accidentally typed `#ifdef`

Comment (kimkulling) [reply code/Common/Exporter.cpp]: Fixed

**Comment by kimkulling** [comment]
Merged, thanks a lot for your contribution.

**Pull request merged**